examples/account/main.tf

```diff
@@ -12,17 +12,39 @@ terraform {
     }
   }
 }
+
 # Include the Tozny Terraform provider for this module
 provider "tozny" {
-
+  api_endpoint = "http://platform.local.tozny.com:8000"
+  username = "terry@forming.com"
+  password = "readymyenvironment"
 }
 
 # A resource for creating a Tozny account
 resource "tozny_account" "my_tozny_account" {
+  save_client_credentials_to_filepath = "./tozny_client_credentials.json"
   profile {
-    id = 1
+    name = "TozFormed"
+    email = "levi+terraform@tozny.com"
+    authentication_salt = "xXp-SXoBEmFA5aU2k50-4g"
```

Comment:
Will these keys be able to be auto-generated at Terraform run time, in a later version?

Reply:
yes